### PR TITLE
added flux

### PR DESCRIPTION
**Pull request by asiyani**

### CODE REVIEW

File: flux/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Flux
+Open and extensible continuous delivery solution for Kubernetes.
+
+flux documents are available [here](https://fluxcd.io/docs)
+
+only source controller is enabled for [Cluster sync from git](https://fluxcd.io/docs/flux-e2e/#diagram-cluster-sync-from-git). Helm and Image automation controllers are disabled.
+
```
Comment (george-angel) [inline flux/README.md:7]: Who are these empty lines for? :)

File: flux/base/00-remove-objects-patch.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+$patch: delete
```
Comment (george-angel) [inline flux/base/00-remove-objects-patch.yaml:1]: Why are the patch files indexed?

File: flux/base/04-roleBinding-patch.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (george-angel) [inline flux/base/04-roleBinding-patch.yaml:1]: No need for new doc separator

File: flux/base/kustomization.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: sys-flux
+
+bases:
```
Comment (george-angel) [inline flux/base/kustomization.yaml:6]: `bases` is deprecated, use `resources`

File: flux/base/00-remove-objects-patch.yaml (hunk 2 above)
Comment (asiyani) [reply flux/base/00-remove-objects-patch.yaml:1]: to keep them in order can remove this in next PR.

**Comment by asiyani** [comment]
sorry closing this PR as there are lots of changes pending.

**Pull request closed without merging**